components/Times.js

```diff
@@ -1,52 +1,84 @@
 import React from "react";
-import { View, Text, StyleSheet, Button, TouchableOpacity } from "react-native";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import normalize from 'react-native-normalize';
+import { Button } from 'react-native-elements';
 
 const Times = ({item, deleteTimer}) => {
   return (
+    <View style={styles.viewContainer}>
       <TouchableOpacity style={styles.listItem} >
         <View style={styles.listItemView}>
-          <Button title='X' style={styles.closeButton} onPress={() => deleteTimer(item.id)}/>
-          <Text style={styles.listItemName}>Name: {item.name}</Text>
-          <Text style={styles.listItemName}>Time: {new Date(item.time).getHours().toString()}:{new Date(item.time).getMinutes().toString()}</Text>
-          <Text style={styles.listItemName}>Window: {new Date(item.window).getHours().toString()}:{new Date(item.window).getMinutes().toString()}</Text>
-          <Text style={item.response === "Answered" ? styles.answered : styles.unanswered}>Status: {item.response === "Answered" ? "Checked In" : "Not Checked In"}</Text>
+          <View style={styles.closeButtonContainer}>
+            <Button type="outline" raised="true" title='X' buttonStyle={styles.closeButton} titleStyle={styles.closeButtonText} onPress={() => deleteTimer(item.id)}/>
+          </View>
+          <Text style={item.response === "Answered" ? styles.answered : styles.unanswered}><Text style={styles.listItemTitle}>Status: </Text>{item.response === "Answered" ? "Checked In" : "Not Checked In"}</Text>
```

Comment:
I do know @D-Lessenden had mentioned being able to conditionally render for not only "Answered", but also "Pending" and "Unanswered" (or the third answer. This is just to make note of where this would be conditionally rendered (perhaps in a function defined elsewhere)